scalafix/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -132,13 +136,55 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
           close <- ctx.matchingParens.close(open.asInstanceOf[Token.LeftBracket])
         } yield
           ctx.replaceToken(open, "(") +
-            ctx.replaceToken(close, ")")).getOrElse(Patch.empty)
+            ctx.replaceToken(close, ")")
+        ).asPatch
+    }.asPatch
+
+  def replaceTupleZipped(ctx: RuleCtx) =
+    ctx.tree.collect {
+      case tupleZipped(Term.Select(Term.Tuple(args), name)) =>
+        val removeTokensPatch =
+          (for {
+            zipped <- name.tokens.headOption
+            closeTuple <- ctx.tokenList.leading(zipped).find(_.is[Token.RightParen])
+            openTuple <- ctx.matchingParens.open(closeTuple.asInstanceOf[Token.RightParen])
+            maybeDot = ctx.tokenList.slice(closeTuple, zipped).find(_.is[Token.Dot])
+          } yield {
+            ctx.removeToken(openTuple) +
+              maybeDot.map(ctx.removeToken).asPatch +
+              ctx.removeToken(zipped)
+          }).asPatch
+
+        def removeSurroundingWhiteSpaces(tk: Token) =
+          (ctx.tokenList.trailing(tk).takeWhile(_.is[Token.Space]).map(ctx.removeToken) ++
+            ctx.tokenList.leading(tk).takeWhile(_.is[Token.Space]).map(ctx.removeToken)).asPatch
+
+        val commas =
+          for {
+            (prev, next) <- args.zip(args.tail)
+            tokensBetweenArgs = ctx.tokenList.slice(prev.tokens.last, next.tokens.head)
+            comma <- tokensBetweenArgs.find(_.is[Token.Comma])
+          } yield comma
+
+        val replaceCommasPatch = commas match {
+          case head :: tail =>
+            ctx.replaceToken(head, ".lazyZip(") +
+              removeSurroundingWhiteSpaces(head) ++
+              tail.map { comma =>
+                ctx.replaceToken(comma, ").lazyZip(") +
+                  removeSurroundingWhiteSpaces(comma)
+              }
+          case _ => Patch.empty
+        }
+
+        removeTokensPatch + replaceCommasPatch
     }.asPatch
 
```

Comment:
Handling comments make the rewrite a lot more complicated compared to the previous version :sweat_smile:.
Note that in lines 153-155 I'm preserving the original closing `)` in order to allow comments on the left side of the last tuple element to be enclosed by the correct parenthesis. Similar care needs to be taken in regards to comments on the left side of a middle triple element (that's the reason for the additional lines in line 174-175).